src/collections/news/2019-11-23-KubeCon NA: CNCF SIG Network Set Intro and Deep-Dive/index.mdx

```diff
@@ -0,0 +1,23 @@
+--
+title: "KubeCon NA: CNCF SIG Network Set Intro and Deep-Dive"
+subtitle: ""
+date: 2019-11-23 12:15:05 +0000
+author: the Newsroom
+thumbnail: ./KubeCon_NA_2019_bg-01.svg
+eurl: https://landscape.cncf.io/category=cncf-members&format=logo-mode&grouping=category&selected=layer5-member&embed=yes
```

Comment:
This should be blank right?

Reply:
I couldn't understand. I had followed the News Style Guide.

Reply:
Yes! But for this particular article, the eurl is null. So you can leave it as `eurl: ""`

Reply:
oh okay